packages/trader/src/AppV2/Components/ServiceErrorSheet/service-error-sheet.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { ActionSheet, Text } from '@deriv-com/quill-ui';
+import { getLanguage, Localize } from '@deriv/translations';
+import { redirectToLogin } from '@deriv/shared';
+import { useSignupHandler } from 'AppV2/Hooks/useSignUpHandler';
+import './service-error-sheet.scss';
```

Comment:
Do we need to import css here if it was imported in index.ts?

Reply:
may need to change import path for ServiceErrorSheet if we remove .scss